static/app/views/acceptOrganizationInvite.tsx

```diff
@@ -135,19 +135,14 @@ class AcceptOrganizationInvite extends AsyncView<Props, State> {
           <ActionsLeft>
             {inviteDetails.hasAuthProvider && (
               <Button
-                aria-label="sso-login"
                 priority="primary"
                 href={this.makeNextUrl(`/auth/login/${inviteDetails.orgSlug}/`)}
               >
                 {t('Join with %s', inviteDetails.ssoProvider)}
               </Button>
             )}
             {!inviteDetails.requireSso && (
-              <Button
-                aria-label="create-account"
-                priority="primary"
-                href={this.makeNextUrl('/auth/register/')}
-              >
+              <Button priority="primary" href={this.makeNextUrl('/auth/register/')}>
```

Comment:
good call!

Reply:
Looks like we needed some of these test IDs for the acceptance tests unfortunately 😢